openeo/extra/job_management.py

```diff
@@ -93,6 +98,10 @@ def __init__(
                 - get_job_dir
                 - get_error_log_path
                 - get_job_metadata_path
+
+        :param max_running_duration [seconds]:
```

Comment:
About the name of this parameter: users will use it like:
```python
manager = MultiBackendJobManager(max_running_duration=1000)
```

This might give wrong impression that the whole manager will stop running after 1000 seconds.

proposal: name this parameter `cancel_job_after` or `cancel_running_job_after` (the latter leaves room to add `cancel_started_job_after` later if we want that)

Reply:
Good suggestion